### PR TITLE
 [android] Add content description for search toolbar & map zoom buttons

### PR DESCRIPTION
**Pull request by Arthur-GYT**
All is in the title ^^.

### CODE REVIEW

**Comment by Arthur-GYT** [comment]
I have a lot of trouble squashing commits...

**Comment by biodranik** [comment]
```
git rebase -i master
# Remove line with regeneration commit
# Change `pick` word to `fixup` or just `f` for every commit in the list except the top one
# Save the file and exit the text editor.

**Comment by biodranik** [comment]
And do not merge master branch to your PR, it complicates the rebase.

**Comment by Arthur-GYT** [comment]
@biodranik It's done ^^.

**Comment by biodranik** [comment]
Looks like you squashed the regeneration commit into your change. It should be a separate commit. We do it to avoid merge conflicts and for easier rebasing (regeneration commit can always be easily removed, and then regenerated again by the script).

**Comment by Arthur-GYT** [comment]
@biodranik I think it's done

File: android/app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -763,15 +763,15 @@
 	<!-- App Tip #00 -->
 	<string name="app_tip_00">Merci d\'utiliser nos cartes créées par la communauté !</string>
 	<!-- App tip #01 -->
-	<string name="app_tip_01">Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !</string>
+	<string name="app_tip_01">Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !</string>
```
Comment (biodranik) [inline android/app/src/main/res/values-fr/strings.xml:766]: Should this go to the last regen commit? You can mark this commit as "edit" during the interactive rebase, and then amend it by removing this change from this commit, and committing it separately. Then squashing it with the following regen commit. Or just remove this change, remove regen commit, finish rebase and regenerate strings again.

Sorry for the hassle, but we need a clean git history, and the skill of editing and amending local commits is required...

File: android/app/src/main/res/values-el/strings.xml [hunk 2]
```diff
@@ -800,6 +800,10 @@
 	<string name="pref_tts_open_system_settings">Ρυθμίσεις συστήματος σύνθεσης ομιλίας</string>
 	<!-- toast displayed when pressing the "Speech synthesis system settings" button, and the system settings aren't found. -->
 	<string name="pref_tts_no_system_tts">Δεν βρέθηκαν οι ρυθμίσεις σύνθεσης ομιλίας, είστε σίγουροι ότι η συσκευή σας την υποστηρίζει;</string>
+	<string name="clear_the_search">Καθαρίστε την αναζήτηση</string>
+	<string name="zoom_in">Μεγέθυνση</string>
+	<!-- Accessibility text for the zoom-out button on the map -->
+	<string name="zoom_out">Μεγέθυνση</string>
```
Comment (cryoranger) [inline android/app/src/main/res/values-el/strings.xml:806]: Σμίκρυνση

**Comment by Arthur-GYT** [comment]
@biodranik Done ^^

File: data/strings/strings.txt [hunk 3]
```diff
@@ -29869,3 +29869,141 @@
...
+    it = Cancella la ricerca
+    ja = 検索をクリアする
+    ko = 검색 지우기
+    lt = Išvalyti paiešką
+    mr = शोध साफ करा
+    nb = Fjern søket
+    nl = Wis de zoekopdracht
+    pl = Wyczyść wyszukiwanie
+    pt = Limpa a pesquisa
+    pt-BR = Limpar a pesquisa
```
Comment (matheusgomesms) [inline data/strings/strings.txt:29906]:
```suggestion
    pt = Limpar a pesquisa
```

File: data/strings/strings.txt [hunk 4]
```diff
@@ -29869,3 +29869,141 @@
...
+    id = Perkecil
+    it = Rimpicciolisci
+    ja = ズームアウト
+    ko = 축소
+    lt = Padidinti mastelį
+    mr = झूम कमी करा
+    nb = Zoom ut
+    nl = Uitzoomen
+    pl = Pomniejsz
+    pt = Reduzir o zoom
```
Comment (matheusgomesms) [inline data/strings/strings.txt:29998]:
```suggestion
    pt = Diminuir o zoom
```

File: android/app/src/main/res/values-pt/strings.xml [hunk 5]
```diff
@@ -768,6 +768,10 @@
 	<string name="pref_tts_open_system_settings">Definições do sistema de síntese de voz</string>
 	<!-- toast displayed when pressing the "Speech synthesis system settings" button, and the system settings aren't found. -->
 	<string name="pref_tts_no_system_tts">Não foram encontradas definições de síntese de voz. Tens a certeza de que o teu dispositivo é compatível?</string>
+	<string name="clear_the_search">Limpa a pesquisa</string>
+	<string name="zoom_in">Aumenta o zoom</string>
+	<!-- Accessibility text for the zoom-out button on the map -->
+	<string name="zoom_out">Reduzir o zoom</string>
```
Comment (matheusgomesms) [inline android/app/src/main/res/values-pt/strings.xml:774]:
```suggestion
	<string name="clear_the_search">Limpar a pesquisa</string>
	<string name="zoom_in">Ampliar</string>
	<!-- Accessibility text for the zoom-out button on the map -->
	<string name="zoom_out">Diminuir o zoom</string>
```

**Comment by Arthur-GYT** [comment]
@matheusgomesms It's done ^^.

**Comment by Jean-BaptisteC** [comment]
Can you rebase your branch to fix conflicts?

**Comment by Arthur-GYT** [comment]
@Jean-BaptisteC The rebase is done ^^.

File: data/strings/strings.txt [hunk 6]
```diff
@@ -29911,3 +29911,140 @@
...
+    id = Perkecil
+    it = Rimpicciolisci
+    ja = ズームアウト
+    ko = 축소
+    lt = Padidinti mastelį
+    mr = झूम कमी करा
+    nb = Zoom ut
+    nl = Uitzoomen
+    pl = Pomniejsz
+    pt = Diminuir o zoom<<<<<<< HEAD
```
Comment (matheusgomesms) [inline data/strings/strings.txt:30039]: Is this correct?

Comment (Arthur-GYT) [reply data/strings/strings.txt:30039]: Oops, it's fixed ^^

**Review by biodranik — APPROVED** [review]
Thank you, and sorry for the delay!

**Pull request merged**